### PR TITLE
feat: add metadata for societesavoie and woposs (DEV-1668)

### PR DESCRIPTION
**Pull request by danielasubotic**

### CODE REVIEW

File: services/metadata/backend/data/societesavoie.json [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
...esse et les enfants du couple princier disposaient de leur propre Hôtel. \nCette recherche porte sur le temps long, du XIIIe au XVe siècle, période décisive pour la genèse de l’État moderne. Il s’agit ainsi de reconstruire, pour la première fois de manière globale, par le biais de sources comptables, normatives et notariales, l’organisation des Hôtels de Savoie, ainsi que leur lente évolution en une institution réglementée, hiérarchisée et codifiée. Les coûts des Hôtels ainsi que leur impact sur les finances savoyardes seront également définis. Le deuxième objectif vise à réaliser une base de données prosopographique des membres des Hôtels. Ceux-ci n’étaient pas qu’une institution : ils étaient composés d’hommes, de femmes et d’enfants appartenant à une microsociété dont les contours précis restent encore à définir. Cette société de cour est abordée dans son ensemble, afin d’apporter une contribution essentielle à la sociologie des élites et de mettre en évidence le poids des réseaux de solidarité et des alliances au sein de la cour de Savoie, mais également de reconstituer des carrières et de saisir l’ascension des officiers et des serviteurs ne faisant pas partie de la noblesse."
```
Comment (BalduinLandolt) [inline services/metadata/backend/data/societesavoie.json:24]: there are these `\n`s in this string. Do they render correctly in the metadata browser locally? If not, you could try `<br/>` instead, but that would be a bit hacky again.

File: services/metadata/backend/data/societesavoie.json [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
...perties, describing the information collected on the individuals belonging to or surrounding the princely households of the court of Savoy (names, gender, functions, time of service, office, income...), as well as the family relationships that may exist between these persons. This information is drawn from accounting, normative and notarial sources, whose archival classifications are always indicated, so that the documents can be traced back.  Finally, bibliographic data can be added to provide additional details on persons and functions.     L’ontologie du projet comprend 18 classes et 157 propriétés, décrivant les informations récoltées sur les individus faisant partie des hôtels princiers de la cour de Savoie ou de leur entourage (noms, genre, fonctions, temps de service, office, revenus...), ainsi que les relations familiales qui peuvent exister entre ces personnes.  Ces informations sont tirées de sources comptables, normatives et notariales, dont la cote d’archive est toujours indiquée, afin de pouvoir remonter aux documents.  Finalement, des données bibliographiques peuvent être ajoutées, afin de fournir des renseignements supplémentaires sur des personnes et des fonctions."
```
Comment (BalduinLandolt) [inline services/metadata/backend/data/societesavoie.json:121]: There are some double spaces here. Should those be newlines? Or just a single space?

File: services/metadata/backend/data/woposs.json [hunk 3]
```diff
@@ -0,0 +1,492 @@
...
+            "Dell’Oro, Francesca; Rimaz, Loris; Bermúdez Sabel, Helena; Marongiu, Paola (2020). Pygmalion-modal 1.0. A tool to draw interactive and diachronic maps of modality . WoPoss. A World of Possibilities. Swiss National Science Foundation.",
+            "Dell'Oro, Francesca; Bermúdez Sabel, Helena; Marongiu, Paola & Reymond, Séverine (2022). Sample of the WoPoss corpus (28th July 2022). Zenodo. Swiss National Science Foundation. DOI: 10.5281/zenodo.6925274.",
+            "Marongiu, Paola & Dell'Oro, Francesca. Interactive diachronic maps of the WoPoss modal markers. Schweizerischer Nationalfonds zur Förderung der wissenschaftlichen Forschung - SNF, Historical linguistics – Institut des sciences du langage - ISLa. Distributed by SWISSUbase, Lausanne, 2022. DOI: 10.48656/qk1e-kk67.",
+            "Dell’Oro, Francesca (2021). “Annotating (ancient) texts and learning (ancient) languages: a possible connection”. LexTech Institute. October 5, 2021."
```
Comment (BalduinLandolt) [inline services/metadata/backend/data/woposs.json:75]: Not really our problem, but the quotation marks here are a bit all over the place

**Review by BalduinLandolt — APPROVED** [review]
Some minor details I remarked on, apart from that it looks good.

Another thing: May I suggest changing the PR title from "add 2 new metadata and some changes" to something like "add metadata for societesavoie and woposs", so that the release notes will reflect which projects have been added?

File: services/metadata/backend/data/societesavoie.json (hunk 1 above)
Comment (danielasubotic) [reply services/metadata/backend/data/societesavoie.json:24]: thx for the hint!
both did not work, so I omitted both options

File: services/metadata/backend/data/woposs.json (hunk 3 above)
Comment (danielasubotic) [reply services/metadata/backend/data/woposs.json:75]: yeah, I know

File: services/metadata/backend/data/societesavoie.json (hunk 2 above)
Comment (danielasubotic) [reply services/metadata/backend/data/societesavoie.json:121]: That is what the conversion did to the old file. I just left it how it was.

**Pull request merged**